src/components/RecipeLinks.astro

```diff
@@ -0,0 +1,76 @@
+---
+import { getEntryBySlug } from 'astro:content';
+
+export interface Props {
+	slugs: string[];
+}
+
+function convertSlug(slug: string) {
+	let corrected = slug;
+	if (slug.startsWith('/')) {
+		corrected = corrected.slice(1);
+	}
+	if (slug.endsWith('/')) {
+		corrected = corrected.slice(0, -1);
+	}
+	return corrected;
+}
+
+const slugs = Astro.props.slugs.map(convertSlug);
+
+const entries = await Promise.all(
+	slugs.map(async (slug, i) => {
+		let entry = await getEntryBySlug('docs', slug);
+		if (!entry) throw Error(`Could not find entry for slug "${Astro.props.slugs[i]}"`);
+		return entry;
+	})
+);
+
+const isList = entries.length > 1;
+const noun = isList ? 'recipes' : 'recipe';
+---
+
+<div class="root">
+	<div class="flex">
+		<img src="/houston_chef.png" width="26" alt="Chef Houston" />
```

Comment:
Would love to hear what @Yan-Thomas thinks, but I’m not sure we want “Chef Houston” to be read out here. Imagine a screen reader: “Chef Houston Related recipes: ...” I think we can treat the image as purely decorative:

```suggestion
		<img src="/houston_chef.png" width="26" alt="" />
```